src/gt4py/next/program_processors/runners/dace_iterator/itir_to_sdfg.py

```diff
@@ -214,6 +319,9 @@ def visit_FencilDefinition(self, node: itir.FencilDefinition):
         for param, type_ in zip(node.params, self.param_types):
             self.add_storage(program_sdfg, str(param.id), type_, neighbor_tables)
 
+        if self.tmps:
```

Comment:
Since the usage of the symbol is 60 lines down, I would here make a note about its usage.

Reply:
Good point, code updated.